pkg/httpserver/httpserver.go

```diff
@@ -28,6 +28,11 @@ type HTTPServer struct {
 	server *http.Server
 }
 
+//DebugServer is the object wrapper for OSM's HTTP server class
```

Comment:
nit: space after `//`
```suggestion
// DebugServer is the object wrapper for OSM's HTTP server class
```